### ISSUE TITLE
coreconcept - pipeline diagram

### ISSUE THREAD

**Issue by heckj** [heckj/swiftui-notes#77]
existing ascii art

```
Publisher source       Operator                          Subscriber
+--------------+      +---------------------------+      +--------------+
|        <Output> --> <Input>      map      <Output> --> <Input>        |
|       <Failure> --> <Failure>  function  <Failure> --> <Failure>      |
+--------------+      +---------------------------+      +--------------+

```
https://heckj.github.io/swiftui-notes/#core-publisher-subscriber

**Comment by heckj** [comment]
https://github.com/heckj/swiftui-notes/pull/116

**Comment by heckj** [comment]
resolved with #116
